lib/web/mage/sticky.js

```diff
@@ -41,8 +44,34 @@ define([
 
             if (!isStatic && this.element.is(':visible')) {
                 offset = $(document).scrollTop() - this.parentOffset;
+
+                if (typeof this.options.spacingTop === 'function') {
```

Comment:
You should check to see whether `spacingTop` is a Number, or whether it returns a Number.
```js
var spacingTop = this.options.spacingTop || 0;
if (typeof this.options.spacingTop === 'function') {
  spacingTop = this.options.spacingTop();
}
spacingTop = Number(spacingTop);
if (isNaN(spacingTop)) {
  throw new Error('sticky: spacingTop must be a Number');
}
offset += this.options.spacingTop;
```
See below for more notes.